opensrp-stock/src/main/java/org/smartregister/stock/activity/StockJsonFormActivity.java

```diff
@@ -290,7 +290,10 @@ private int calculateDosesWasted(int childrenVaccinated, int vialsIssued, int wa
     private void displayChildrenVialsUsed(int childrenVaccinated, int vialsUsed, int vialsWasted) {
         try {
             stockJsonFormFragment.getLabelViewFromTag("Children_Vaccinated_Count", String.format(getString(R.string.children_vaccinated), childrenVaccinated));
```

Comment:
Could we move the string literals referencing tags in code to constants?

Reply:
Resolved